Scripts/Modules/WorkspaceModule.cs

```diff
@@ -148,6 +149,9 @@ public void OnDeserializePreferences(object obj)
                 var workspaceType = Type.GetType(workspaceLayout.name);
                 if (workspaceType != null)
                 {
+                    if (HiddenTypes.Contains(workspaceType))
```

Comment:
Why this?

Reply:
I mention this in my comment below.  I found it odd that workspaces from a previous session, which won't be available in this particular context, come up when starting EXR